File: ToDoApp.BAL/DTO/TaskInfo.cs
﻿namespace ToDoApp.BAL.DTO
{
    public class TaskInfo
    {
        public int Id { get; set; }

        public string Title { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public string status {  get; set; } = string.Empty;

        public DateTime CreatedDate { get; set; }

    }
}

File: src/app/core/models/task-info.ts
export class TaskInfo {
    Id: number = 0;
    Title: string = '';
    Description: string = '';
    Status: string = '';
    CreatedDate: Date = new Date();

    constructor(args: any) {
        this.Id = args.id || args.Id || 0;
        this.Title = args.title || args.Title || '';
        this.Description = args.description || args.Description || '';
        this.CreatedDate = args.createdDate || args.CreatedDate || new Date();
        this.Status = args.Status || args.status || '';
    }
}